siliconcompiler/tools/yosys/syn_fpga.py

```diff
@@ -38,10 +38,75 @@ def setup_fpga(chip):
              ",".join(['fpga', part_name, 'lutsize']),
              step=step, index=index)
 
+    chip.add('tool', tool, 'task', task, 'require',
+             ",".join(['fpga', part_name, 'var', 'flop_async_set']),
+             step=step, index=index)
+
+    chip.add('tool', tool, 'task', task, 'require',
+             ",".join(['fpga', part_name, 'var', 'flop_async_reset']),
+             step=step, index=index)
+
+    chip.add('tool', tool, 'task', task, 'require',
+             ",".join(['fpga', part_name, 'var', 'flop_enable']),
+             step=step, index=index)
+
+    chip.add('tool', tool, 'task', task, 'require',
+             ",".join(['fpga', part_name, 'var', 'legalize_flops']),
+             step=step, index=index)
```

Comment:
no need to require the feature flags. But you will need to add help string to it so it's clear what these are in the documentation